.github/workflows/cmake.yml

```diff
@@ -10,9 +10,9 @@ jobs:
         include:
           - os: ubuntu-latest
             name: Linux
-            container: wpilib/roborio-cross-ubuntu:2022-20.04
+            container: wpilib/roborio-cross-ubuntu:2022-18.04
```

Comment:
Is there a reason these containers are changing?

Reply:
Accidentally.